codespell_lib/data/dictionary.txt

```diff
@@ -14911,6 +14931,8 @@ weant->want, wean,
 weaponary->weaponry
 weas->was
 webiste->website
+wedensday->wednesday
```

Comment:
I think we can capitalise the suggestion correctly can't we.

Reply:
@larsoner opinion?

Reply:
yes we can